canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss

```diff
@@ -39,8 +39,9 @@
 		margin-bottom: $spacing-02;
 		margin-right: $spacing-04;
 		display: grid;
-		grid-template-columns: 1fr 8fr;
```

Comment:
![Screen Shot 2022-07-11 at 7 36 51 AM](https://user-images.githubusercontent.com/21126000/178265729-813cea64-3028-475b-9c20-aec0c2fae155.png)

You might need to have another look at this.  This cause longer radio to wrap too early.  I don't think `grid-template-columns` works for this case.